docs/releases/release-support-policy.mdx

```diff
@@ -87,4 +87,5 @@ This page describes Scalar's support policy for major and minor version releases
   </tbody>
 </table>
 
-&#42; This product version is no longer supported under Maintenance Support or Assistance Support.
+\* "TBD" will be replaced with a date after the next minor version is released.<br />
```

Comment:
We will think about how we handle the next major version because 3.x and 4.x will co-exist for a while so their EOL are separately managed.
For 3.x, I think the current one is fine.

Reply:
Thank you for your explanation. Then I am fine.